public class ExceptionTest {
	public static void main(String[] args) {
		
		try {
			Thread.sleep(1000);
			int a = 100/0;
		} catch (InterruptedException e) {
			// TODO Auto-generated catch block
			e.printStackTrace();
		} catch(ArithmeticException e){
			e.printStackTrace();
		}
		
		System.out.println("end");
	}
}
